### ISSUE TITLE
Accessibility testing - Alt tags needed 

### ISSUE THREAD

**Issue by bridgettighe** [Phan717/web-design-2-independent-project#5]
<img width="337" alt="Screen Shot 2021-12-09 at 6 35 56 PM" src="https://user-images.githubusercontent.com/77281280/145492350-176177fe-2f72-4051-b3c3-ef8a4fa8ca15.png">

Problem: Images need to have descriptive alt tags and important elements should have aria labels. When using VoiceOver there wasn't enough descriptiveness. 
Solution: Add alt tags and aria-label tags to content to ensure the website is more accessible.
